src/PullRequest/Application/CommandHandler/CheckMilestoneCommandHandler.php

```diff
@@ -18,11 +18,6 @@ public function __construct(
 
     public function __invoke(CheckMilestoneCommand $command): void
     {
-        // This command is only for PrestaShop/PrestaShop repository
-        if ('PrestaShop' !== $command->repositoryOwner || 'PrestaShop' !== $command->repositoryName) {
```

Comment:
If we simply remove this, we might have the message being prompted even on repositories like https://github.com/PrestaShop/PrestaShop-webservice-lib

I see 2 possibilities:
1. We verify here if repository is either the Core or a module. To know if it's a module we could define it in a config array like https://github.com/PrestaShop/kanbanbot/blob/main/config/services.php#L26
2. We simply verify if repository milestones is not empty (= the repo uses milestones). Need one more API call -> maybe overkill?

Reply:
You're right!

I like the 1, but... it need to be change as we discover some usecases.
For the 2, it's certainly possible to have some old repo that don't use anymore milestones, but they are still present.
(or we filter this query with opened milestone only) 🤔

Reply:
So let's do the solution 2!